specs-go/v1/manifest.go

```diff
@@ -30,6 +30,9 @@ type Manifest struct {
 	// Layers is an indexed list of layers referenced by the manifest.
 	Layers []Descriptor `json:"layers"`
 
+	// Refers is an optional link to any existing manifest within the repository.
+	Refers *Descriptor `json:"refers,omitempty"`
```

Comment:
particular reason for using a pointer here? Looks like we're using a mix, and usually there was reasoning for choosing one way or the other

Reply:
To support `omitempty`. A struct is never "empty" to Go because it can't be compared to a zero value.